util/netcdf_generator.py

```diff
@@ -192,6 +192,9 @@ def _create_files(self, base_path):
 
                 if params_to_include:
                     ds = self._filter_params(ds, params_to_include)
+                # filter the parser set quality_flag parameter (see discussion in ticket 13284)
```

Comment:
I realize the ticket is NetCDF specific, but does this parameter also appear in when data is retrieved in formats other than NetCDF? If so, since this change is specific to the NetCDF packaging process, it will still appear on the other packaging formats.

Wouldn't a better approach be to not create the parameter rather than creating it and then filtering it out?

Reply:
Yes. I originally made the change this way because I did not know how to prevent stream engine from including that parameter when fetching the data from Cassandra. Since then I found a config list that will exclude it. Making updates...